examples/c/example.c

```diff
@@ -175,42 +167,42 @@ AranyaError init_client(Client *c, const char *name, const char *daemon_addr,
         c->pk = realloc(c->pk, c->pk_len);
         err   = aranya_get_key_bundle(&c->client, c->pk, &c->pk_len);
     }
-    CLIENT_EXPECT("error getting key bundle", c->name, err);
+    CLIENT_EXPECT(err, "error getting key bundle", c->name);
 
     return ARANYA_ERROR_SUCCESS;
 }
 
-// Initialize the Aranya `Team` by first initializing the team's clients and
-// then creates the team.
+// Initializes an Aranya `Team`, initializing each client and creating the team.
 AranyaError init_team(Team *t) {
     AranyaError err;
 
     // initialize team clients.
     for (int i = 0; i < NUM_CLIENTS; i++) {
         printf("initializing client: %s\r\n", client_names[i]);
-        err = init_client(&t->clients_arr[i], client_names[i], daemon_socks[i],
-                          aqc_addrs[i]);
-        EXPECT("error initializing team", err);
+        EXPECT(init_client(&t->clients_arr[i], client_names[i], daemon_socks[i],
+                           aqc_addrs[i]),
+               "error initializing `Client`");
     }
 
-    // have owner create the team.
-    // The `aranya_create_team` method is used to create a new graph for the
-    // team to operate on.
-    AranyaTeamConfigBuilder build;
-    AranyaTeamConfig cfg;
-    aranya_team_config_builder_build(&build, &cfg);
-    err = aranya_create_team(&t->clients.owner.client, &cfg, &t->id);
-    EXPECT("error creating team", err);
+    // Have the owner call `aranya_create_team`, which creates a new graph for
+    // the team to use.
+    AranyaTeamConfigBuilder team_build;
+    AranyaTeamConfig team_cfg;
+    EXPECT(aranya_team_config_build(&team_build, &team_cfg),
+           "error building `TeamConfig`");
+    EXPECT(aranya_create_team(&t->clients.owner.client, &team_cfg, &t->id),
+           "error creating aranya team");
 
     // Test ID serialization and deserialization
     size_t team_id_str_len = ARANYA_ID_STR_LEN;
     char *team_id_str      = malloc(team_id_str_len);
-    aranya_id_to_str(&t->id.id, team_id_str, &team_id_str_len);
+    EXPECT(aranya_id_to_str(&t->id.id, team_id_str, &team_id_str_len),
+           "error converting ID to string");
     printf("Team ID: %s \r\n", team_id_str);
 
     AranyaId decodedId;
-    err = aranya_id_from_str(team_id_str, &decodedId);
-    EXPECT("error decoding string into an ID", err);
+    EXPECT(aranya_id_from_str(team_id_str, &decodedId),
```

Comment:
Should call this variable `decoded_id`:
```suggestion
    EXPECT(aranya_id_from_str(team_id_str, &decoded_id),
```